packages/edit-post/src/components/layout/style.scss

```diff
@@ -64,6 +64,10 @@
 
 .has-metaboxes .editor-visual-editor {
 	flex: 1;
+
+	&.is-iframed {
+		isolation: isolate;
+	}
```

Comment:
@t-hamano given how this relates to #63939 and it be great if you can confirm the rule seems safe to apply. Now it’s only done if there are metaboxes and the editor is iframed whereas the last time it’d be done unconditionally.

Reply:
I believe this rule should be safe